content/tracing/advanced_usage/manual_tracing.md

```diff
@@ -33,6 +33,76 @@ public class MyClass {
 
 {{% /tab %}}
 {{% tab "Python" %}}
+
+If you would like to extend the functionality of the ``ddtrace`` library or gain
+finer control over instrumenting your application, several techniques are
+provided by the library.
+
+The following examples use the global tracer object which can be imported via:
+
+```python
+  from ddtrace import tracer
+```
+
+**Decorator**
+
+``ddtrace`` provides a decorator that can be used to trace a particular method
+in your application:
+
+```python
+  @tracer.wrap()
+  def business_logic():
+    """A method that would be of interest to trace"""
+    # ...
+    # ...
+```
+
+API details for the decorator can be found at [`ddtrace.Tracer.wrap()`][py_wrap]
+
+**Context Manager**
+
+To trace an arbitrary block of code, you can use the [`ddtrace.Span`][py_span]
+context manager:
+
+```python
+  import time
+
+  # trace some sleeping
+  with tracer.trace('time.sleep'):
+    # do interesting stuff besides sleeping for a second
```

Comment:
`# do some time expensive operation` or similar but not "stuff". Also don't use `time.sleep(1)`. Use the following as you did in the previous snippet for consistency:
```python
# ...
# ...
```